examples/es5_output/es5_output_test.sh

```diff
@@ -1,40 +1,69 @@
 #!/bin/bash
 set -e
 
+MANIFEST="$TEST_SRCDIR/MANIFEST"
+if [ -e "$MANIFEST" ]; then
+  while read line; do
+    declare -a PARTS=($line)
+    if [ "${PARTS[0]}" == "build_bazel_rules_typescript/examples/some_library/library.js" ]; then
+      readonly LIBRARY_JS=$(cat ${PARTS[1]})
+    elif [ "${PARTS[0]}" == "build_bazel_rules_typescript/examples/bar.js" ]; then
+      readonly BAR_JS=$(cat ${PARTS[1]})
+    elif [ "${PARTS[0]}" == "build_bazel_rules_typescript/examples/foo.js" ]; then
+      readonly FOO_JS=$(cat ${PARTS[1]})
+    fi
+  done < $MANIFEST
+else
+  readonly LIBRARY_JS=$(cat $TEST_SRCDIR/build_bazel_rules_typescript/examples/some_library/library.js)
+  readonly BAR_JS=$(cat $TEST_SRCDIR/build_bazel_rules_typescript/examples/bar.js)
+  readonly FOO_JS=$(cat $TEST_SRCDIR/build_bazel_rules_typescript/examples/foo.js)
+fi
+
 # should produce named UMD modules
-readonly LIBRARY_JS=$(cat $TEST_SRCDIR/build_bazel_rules_typescript/examples/some_library/library.js)
 if [[ "$LIBRARY_JS" != *"define(\"some-lib/library\""* ]]; then
   echo "Expected library.js to declare named module, but was"
-  echo "$A_JS"
+  echo "$LIBRARY_JS"
+  exit 1
+fi
+
+# should produce named UMD modules
+if [[ "$BAR_JS" != *"define(\"build_bazel_rules_typescript/examples/bar\""* ]]; then
+  echo "Expected bar.js to declare named module, but was"
+  echo "$BAR_JS"
   exit 1
 fi
 
 # should give a name to required modules
-readonly BAR_JS=$(cat $TEST_SRCDIR/build_bazel_rules_typescript/examples/bar.js)
 if [[ "$BAR_JS" != *"require(\"build_bazel_rules_typescript/examples/foo\")"* ]]; then
   echo "Expected bar.js to require named module foo, but was"
   echo "$BAR_JS"
   exit 1
 fi
 
 # should give a name to required modules from other compilation unit
-readonly FOO_JS=$(cat $TEST_SRCDIR/build_bazel_rules_typescript/examples/bar.js)
-if [[ "$FOO_JS" != *"require(\"some-lib/library\")"* ]]; then
+if [[ "$BAR_JS" != *"require(\"some-lib/library\")"* ]]; then
```

Comment:
why did this change from FOO to BAR? seems like a mistake?

Reply:
the original was wrong. code was actually pointing to bar.js: `readonly FOO_JS=$(cat $TEST_SRCDIR/build_bazel_rules_typescript/examples/bar.js)`